src/Hosting/TestHost/src/TestWebSocket.cs

```diff
@@ -337,7 +332,7 @@ private void ThrowNoReceive()
         {
             if (_receiverClosed)
             {
-                throw new ObjectDisposedException(typeof(TestWebSocket).FullName);
+                ObjectDisposedException.ThrowIf(_receiverClosed, typeof(TestWebSocket));
```

Comment:
In the original code, there is an `if` condition wrapping exception, that's why I added there to overload. And also in the new overload, we don't have to pass FullName, under the hood it will be retrieved. 
See https://source.dot.net/#System.Private.CoreLib/ThrowHelper.cs,393